app/core/config.py

```diff
@@ -0,0 +1,108 @@
+"""
+Environments and configuration file
+"""
+import logging
+import os
+import secrets
+from enum import Enum
+from functools import lru_cache
+from typing import Optional
+from pydantic import BaseSettings
+
+logger = logging.getLogger(__name__)
+
+
+class EnvironmentEnum(str, Enum):
+    """
+    Enumerate the application environments
+    """
+
+    PRODUCTION = "production"
+    LOCAL = "local"
```

Comment:
Can this be `DEVELOPMENT = "development"` for consistency or is that not Pythonic? (disclaimer: I haven't kept up with Python for over a year, so I'm not sure what's standard these days 😄 )